grassroots-backend/src/phone-canvass/PhoneCanvass.controller.ts

```diff
@@ -216,4 +220,16 @@ export class PhoneCanvassController {
     session.phoneCanvassCaller = caller;
     return caller;
   }
+
+  @Get("start-simulation/:id")
+  async startSimulation(@Param("id") id: string): Promise<VoidDTO> {
+    if (!(await getEnvVars()).ENABLE_PHONE_CANVASS_SIMULATION) {
+      throw new ForbiddenException(
+        "Can't simulate a phone canvass without ENABLE_PHONE_CANVASS_SIMULATION",
```

Comment:
What is the benefit of this check? It seems like we would want to simulate in all environments.

Reply:
Maybe? We really want to avoid hitting the simulate button on a real phone canvass, especially a partially completed one, as we could lose the state.

I think we could protect against this, but for now, it's kinda dangerous.